lib/chewy/rake_helper.rb

```diff
@@ -268,6 +268,24 @@ def delete_by_query_options_from_env(env)
           end
       end
 
+      def create_missing_indexes!(output: $stdout)
+        subscribed_task_stats(output) do
+          Chewy.eager_load!
+          all_indexes = Chewy::Index.descendants
+          all_indexes = all_indexes - [Chewy::Stash::Journal] unless Chewy.configuration[:journal]
+          Chewy::Index.descendants.each do |index|
```

Comment:
I guess you want to use `all_indexes` here?
```suggestion
          all_indexes.each do |index|
```

Reply:
oops, right.